sevenseconds/config/vpc.py

```diff
@@ -229,10 +232,12 @@ def configure_subnet(vpc, az, _type: str, cidr: IPNetwork, dry_run: bool, waiter
                      'Values': [az]}
                 ])
                 # We are to fast for AWS (InvalidSubnetID.NotFound)
-                subnet.create_tags(Tags=[
-                    {'Key': 'Name',
-                     'Value': name}
-                ])
+                tags.append({'Key': 'Name', 'Value': name})
+    if _type == 'dmz':
+        tags.append({'Key': 'kubernetes.io/role/elb', 'Value': ''})
+        tags.append({'Key': 'kubernetes.io/role/internal-elb', 'Value': ''})
```

Comment:
Should these only be added for 'Kubernetes' accounts e.g. accounts with a special config. Now it seems to be enabled for all accounts with a `dmz` network?

Reply:
This is a good point. We have no difference between stups and kubernetes accounts. I think, it would be no impact, if we set the tags to every account